eng/Configurations.props

```diff
@@ -52,7 +52,8 @@
     <TargetOS Condition="'$(TargetOS)' == '' and $([MSBuild]::IsOSPlatform('OSX'))">OSX</TargetOS>
     <TargetOS Condition="'$(TargetOS)' == '' and $([MSBuild]::IsOSPlatform('FREEBSD'))">FreeBSD</TargetOS>
     <TargetOS Condition="'$(TargetOS)' == '' and $([MSBuild]::IsOSPlatform('NETBSD'))">NetBSD</TargetOS>
-    <TargetOS Condition="'$(TargetOS)' == '' and $([MSBuild]::IsOSPlatform('SUNOS'))">SunOS</TargetOS>
+    <TargetOS Condition="'$(TargetOS)' == '' and $([MSBuild]::IsOSPlatform('ILLUMOS'))">Illumos</TargetOS>
```

Comment:
`Illumos` -> `illumos`, the first `I` is lowercase to avoid confusing with lowercase `L`

Reply:
Thanks. I have fixed capitalization to match official name `illumos` (all letters in lower case). The remaining instances are in MSBuild and C# variable names, which generally follow PascalCase naming convention.